### DISCUSSION TITLE
EAPG Rates on Machine Readable File Requirement

### DISCUSSION THREAD

**Discussion by cgmurphy** [CMSgov/hospital-price-transparency#154] [Q&A] [answered]
Price Transparency Machine Readable file we have several ACO Medicaid contracts where we negotiated a rate, outpatient services are a percentage above Medicaid/MassHealth. Medicaid/ MassHealth is an EPAG pricing, and it is proprietary information under 3M of how the claim is grouped to an EAPG.
Could we list the Algorithm, with a note that the information is proprietary information? Or are required to calculate the EAPG amount where an EAPG amount would apply to the outpatient service, and add it to negotiated Dollar amount on the MRF?
Thank you for your assistance.

**Comment by RidenourK** [comment, marked as answer]
@cgmurphy . Thank you for your question. 

As explained in the CY2024 OPPS/ASC final rule, hospitals use different methods to establish standard charges for items and services, resulting in charge/item and charge/service combinations that are often unique to that hospital. Therefore, although CMS has recently taken steps toward standardization, each hospital will continue to have some discretion related to how it chooses to encode its standard charge information (including information related to payer-specific negotiated charges) within its machine-readable file, so long as the file conforms to the CMS template layouts and data specifications as described at 45 CFR 180.50(c)(2).  

For example, you could list the item/service, indicate the “code type” is EAPG, provide the algorithm in the “Payer-specific Negotiated Charge: Algorithm” field, indicate the “Standard Charge Methodology” is “Other”, describe the algorithm in the notes field and calculate the “estimated allowed amount” for that item/service. Alternatively, you could list the item/service, indicate the “code type” is EAPG, calculate the base rate in the “Payer-specific Negotiated Charge: Dollar” field, indicate the “Standard Charge Methodology” is “Other” and describe what factors may impact the base rate in the notes field. Another approach would be to list the item/service, indicate the “code type” is EAPG, calculate the EAPG amount and add it to the base rate in the “Payer-specific Negotiated Charge: Dollar” field, indicate the “Standard Charge Methodology” is “Other” and describe how the “Payer-specific Negotiated Charge: Dollar” was derived in the notes field.
